idseq_dag/steps/blast_contigs.py

```diff
@@ -29,7 +110,7 @@ class PipelineStepBlastContigs(PipelineStep):
     -out {blast_m8}
     -outfmt 6
```

Comment:
We also need to specify the out columns, similar to blastn
-outfmt '6 qseqid sseqid pident qlen slen length mismatch gapopen qstart qend sstart send evalue bitscore'

Reply:
I was under the impression that we are only testing the changes to the blastn command in this version.  Do we also want to change the blastx command?